### PR TITLE
ETCD-606: Batch bundle revision rollout 

### PR DESCRIPTION
**Pull request by tjungblu**
code from #1268 with vendor of https://github.com/openshift/library-go/pull/1739 for testing

/hold

### CODE REVIEW

**Comment by tjungblu** [comment]
/payload-aggregate periodic-ci-openshift-release-master-ci-4.17-upgrade-from-stable-4.16-e2e-aws-ovn-upgrade 10

**Comment by tjungblu** [comment]
/payload-aggregate periodic-ci-openshift-release-master-ci-4.17-upgrade-from-stable-4.16-e2e-aws-ovn-upgrade 10

**Comment by tjungblu** [comment]
/payload-aggregate periodic-ci-openshift-release-master-ci-4.17-upgrade-from-stable-4.16-e2e-aws-ovn-upgrade 10

**Comment by tjungblu** [comment]
/payload-aggregate periodic-ci-openshift-release-master-ci-4.17-upgrade-from-stable-4.16-e2e-aws-ovn-upgrade 1

**Comment by tjungblu** [comment]
/payload-aggregate periodic-ci-openshift-release-master-ci-4.17-upgrade-from-stable-4.16-e2e-aws-ovn-upgrade 10

File: pkg/operator/starter.go [hunk 1]
```diff
@@ -568,21 +568,30 @@ func RunOperator(ctx context.Context, controllerContext *controllercmd.Controlle
...
+	go etcdEndpointsController.Run(ctx, 1)
+	go clusterMemberController.Run(ctx, 1)
+
 	go masterNodeInformer.Run(ctx.Done())
 	dynamicInformers.Start(ctx.Done())
 	operatorInformers.Start(ctx.Done())
 	kubeInformersForNamespaces.Start(ctx.Done())
 
+	// wait for all caches to sync before we're starting the majority of controllers
+	kubeInformersForNamespaces.Start(ctx.Done())
```
Comment (lance5890) [inline pkg/operator/starter.go:580]: this may be duplicate of line 577

Comment (tjungblu) [reply pkg/operator/starter.go:580]: yep, thanks. It didn't help, so I'm going to remove that commit again :/

**Comment by tjungblu** [comment]
/payload-aggregate periodic-ci-openshift-release-master-ci-4.17-upgrade-from-stable-4.16-e2e-aws-ovn-upgrade 10

**Comment by tjungblu** [comment]
/payload-aggregate periodic-ci-openshift-release-master-ci-4.17-upgrade-from-stable-4.16-e2e-aws-ovn-upgrade 5

**Comment by tjungblu** [comment]
/payload-aggregate periodic-ci-openshift-release-master-ci-4.17-upgrade-from-stable-4.16-e2e-aws-ovn-upgrade 10

**Comment by tjungblu** [comment]
/payload-aggregate periodic-ci-openshift-release-master-ci-4.17-upgrade-from-stable-4.16-e2e-aws-ovn-upgrade 10

**Comment by tjungblu** [comment]
/payload-aggregate periodic-ci-openshift-release-master-ci-4.17-upgrade-from-stable-4.16-e2e-aws-ovn-upgrade 10

**Comment by tjungblu** [comment]
/payload-aggregate periodic-ci-openshift-release-master-ci-4.17-upgrade-from-stable-4.16-e2e-aws-ovn-upgrade 10

**Comment by tjungblu** [comment]
/payload-aggregate periodic-ci-openshift-release-master-ci-4.17-upgrade-from-stable-4.16-e2e-aws-ovn-upgrade 10

**Pull request merged**